content/zh-cn/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -64,11 +71,18 @@ you have selected.
 <!--
 * For version 1.25 and 1.26, enabling the feature via kube-apiserver feature gate is required.
 Set `--feature-gates=KMSv2=true` to configure a KMS v2 provider.
+ For environments where all API servers are running version 1.28 or later, and you do not require the ability
+ to downgrade to Kubernetes v1.27, you can enable the `KMSv2KDF` feature gate (a beta feature) for more
+ robust data encryption key generation. The Kubernetes project recommends enabling KMS v2 KDF if those
+ preconditions are met.
 
 * Your cluster must use etcd v3 or later
 -->
 * 对于 Kubernetes 1.25 和 1.26 版本，需要通过 kube-apiserver 特性门控启用此特性。
   设置 `--feature-gates=KMSv2=true` 以配置 KMS v2 驱动。
+  对于所有 API 服务器正在运行 1.28 或更高的版本并且你不需要降级到 Kubernetes v1.27 的环境，
+  你可以启用 `KMSv2KDF` 特性门控（Beta 特性）以生成更稳健的数据加密密钥。
+  如果这些前提条件均已满足，则 Kubernetes 项目建议启用 KMS v2 KDF。
```

Comment:
Could write 密钥派生函数 rather than KDF. Or 密钥派生函数 and also explain the relevance English abbreviation.

Eg, something roughly like:
```markdown
…密钥派生函数 。用英文，你写 _key derivation function_ （KDF） 。